src/pages/about-us/index.js

```diff
@@ -1,14 +1,63 @@
 import React from 'react'
 import styled from 'styled-components'
 import { localize, WithIntl } from 'components/localization'
-import { SEO } from 'components/containers'
 import Layout from 'components/layout/layout'
+import { SEO, SectionContainer } from 'components/containers'
+import { Header } from 'components/elements'
+import device from 'themes/device'
 
-const StyledDiv = styled.div`
-    font-size: 330%;
+const StyledFirstSectionText = styled.p`
```

Comment:
```suggestion
const StyledFirstSectionText = styled.p`
```
you can use the `Text` component from `typography.js` for this and pass props instead of creating a new styled component

Reply:
done